features/support/world.js

```diff
@@ -25,12 +25,12 @@ class CucumberElectronWorld {
     args[0] = args[0].replace(/^cucumber-electron/, 'cucumber-electron.js')
     args[0] = path.resolve(__dirname + '/../../bin/' + args[0])
 
-    this.execResult = { stdout: '', stderr: '', output: '', exitCode: null, error: null }
+    this.execResult = { stdout: '', stderr: '', exitCode: null, error: null }
     this.printExecResult = () =>
       '------------------------------------\n' +
       `The process exited with code ${this.spawnedProcess.exitCode}\n` +
       '------------------------------------\n' +
-      `OUTPUT:\n${this.execResult.output}\n` +
+      `Exception:\n${(this.execResult.error && this.execResult.error.stack) || ''}\n` +
```

Comment:
Could we make this not appear at all if there is no exception, to avoid noise?

Reply:
I'd prefer to keep it since it sometimes helps diagnosing weird errors.